p2p-specs/p2p-interface.md

```diff
@@ -202,6 +213,11 @@ minimumStake: '0.0'
 ```
 The `mempool-id` of the canonical mempool is `TBD` (IPFS hash of the yaml/JSON file).
 
+#### Canonical Mempools
+
+There will be a published list of canonical mempools maintained by the bundler community. This list represents mempools that support the full [ERC-7562](https://github.com/ethereum/ERCs/pull/105) validation rules as well as certain mempool configuration parameters and a specific entry point contract. All bundlers SHOULD support these mempools. User operations that do not require access to alternative mempools will be supported by at least one of these canonical mempools.
```

Comment:
Can we pls link to this file? https://github.com/eth-infinitism/bundler-spec/pull/22/files